rtt_roscomm/include/rtt_roscomm/rtt_rostopic_ros_msg_transporter.hpp

```diff
@@ -264,9 +266,9 @@ namespace rtt_roscomm {
         RTT::log(RTT::Debug)<<"Creating ROS subscriber for port "<<port->getName()<<" on topic "<<policy.name_id<<RTT::endlog();
       }
       if(topicname.length() > 1 && topicname.at(0) == '~') {
-        ros_sub = ros_node_private.subscribe(policy.name_id.substr(1), policy.size > 0 ? policy.size : 1, &RosSubChannelElement::newData, this); // minimum queue_size 1
+        ros_sub = subscribe(ros_node_private, &prassthrough_callback_queue, policy.name_id.substr(1), policy.size > 0 ? policy.size : 1, &RosSubChannelElement::newData, this); // minimum queue_size 1
       } else {
-        ros_sub = ros_node.subscribe(policy.name_id, policy.size > 0 ? policy.size : 1, &RosSubChannelElement::newData, this); // minimum queue_size 1
+        ros_sub = subscribe(ros_node, &prassthrough_callback_queue, policy.name_id, policy.size > 0 ? policy.size : 1, &RosSubChannelElement::newData, this); // minimum queue_size 1
       }
     }
```

Comment:
Suggestion without the need for any of the `rtt_roscomm::subscribe()` template methods defined in `passthrough_callback_queue.hpp`:

```cpp
    RosSubChannelElement(RTT::base::PortInterface* port, const RTT::ConnPolicy& policy) :
      ros_node(),
      ros_node_private("~")
    {
      ros::SubscribeOptions ops;
      ops.initByFullCallbackType<T>(
          policy.name_id,
          policy.size > 0 ? policy.size : 1,  // minimum queue_size 1
          boost::bind(&RosSubChannelElement::newData, this, _1));
      ops.callback_queue = &passthrogh_callback_queue;

      RTT::Logger::In in(policy.name_id);
      if (port->getInterface() && port->getInterface()->getOwner()) {
        RTT::log(RTT::Debug) << "Creating ROS subscriber for port " << port->getInterface()->getOwner()->getName() << "." << port->getName() << " on topic " << ops.topic << RTT::endlog();
      } else {
        RTT::log(RTT::Debug) << "Creating ROS subscriber for port " << port->getName() << " on topic " << ops.topic << RTT::endlog();
      }

      if (ops.topic.length() > 1 && ops.topic[0] == '~') {
        ops.topic = ops.topic.substr(1);
        ros_sub = ros_node_private.subscribe(ops);
      } else {
        ros_sub = ros_node.subscribe(ops);
      }
    }
```